CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Accepted into CNCF:
 - [#1297](https://github.com/thanos-io/thanos/pull/1297) Thanos Compact added `/-/ready` and `/-/healthy` endpoints to Thanos compact.
 - [#1431](https://github.com/thanos-io/thanos/pull/1431) Thanos Query added hidden flag to allow the use of downsampled resolution data for instant queries.
 - [#1408](https://github.com/thanos-io/thanos/pull/1408) Thanos Store Gateway can now allow the specifying of supported time ranges it will serve (time sharding). Flags: `min-time` & `max-time`
+- [#1427](https://github.com/thanos-io/thanos/pull/1427) Added "Status" tab to Ruler UI
```

Comment:
It has to be part of `Unreleased` now ):

Reply:
👍 fixed :)